package.json

```diff
@@ -3,15 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "codemirror": "^5.32.0",
     "es6-promise": "^4.1.0",
     "history": "^4.7.2",
     "isomorphic-fetch": "^2.2.1",
     "normalize.css": "^7.0.0",
     "react": "^15.6.1",
-    "react-codemirror": "^1.0.0",
+    "react-codemirror2": "^3.0.7",
```

Comment:
I had to use another fork for code mirror component, because original one contains bug and maintainer doesn't support project anymore https://github.com/JedWatson/react-codemirror/issues/106